src/app/parcels/ActionBar/Actions.tsx

```diff
@@ -149,6 +161,7 @@ const ActionsButton: React.FC<ActionsButtonProps> = ({
 
 interface Props {
     selected: number[];
+    setSelected: React.Dispatch<React.SetStateAction<number[]>>;
```

Comment:
shall we rename these to `selectedRowIndices`  / `setSelectedRowIndices` or something? It's quite hard to understand what this array of numbers is.

Reply:
Have done